packages/core/core/src/Parcel.js

```diff
@@ -115,6 +117,23 @@ export default class Parcel {
       this.#initialOptions,
     );
     this.#resolvedOptions = resolvedOptions;
+
+    if (resolvedOptions.featureFlags.parcelV3) {
+      // eslint-disable-next-line no-unused-vars
+      let {entries, inputFS, outputFS, ...options} = this.#initialOptions;
+
+      this.#parcelV3 = new ParcelV3({
```

Comment:
Nitpick, but do we really need to store on `this`? Looks like it could be local to this method?

Reply:
I can make it local, I think I had it designed a different way initially where it was necessary then